code/modules/martial_arts/unathi.dm

```diff
@@ -45,8 +45,22 @@
 	add_to_streak("D",D)
 	if(check_streak(A,D))
 		return 1
-	basic_hit(A,D)
-	return 1
+	if(istype(D, /mob/living/simple_animal))
+		simple_animal_basic_disarm(A,D)
+		return 1
+	else
+		basic_hit(A,D)
+		return 1
+
+/datum/martial_art/kis_khan/simple_animal_basic_disarm(var/mob/living/carbon/human/A, var/mob/living/simple_animal/D)
+	if(!D.paralysis)
+		A.visible_message("<b>[A]</b> delivers a blow to \the <b>[D]</b>'s head, making [D.get_pronoun("him")] fall unconscious!")
+		A.do_attack_animation(D)
+		playsound(D.loc, /singleton/sound_category/punch_sound, 25, 1, -1)
```

Comment:
This isn't an acceptable form of playsound, respect the header:

`playsound(get_turf(D), /singleton/sound_category/punch_sound, 25, TRUE, ONE_OF_THE_DEFINES_THAT_ENDS_WITH_SOUND_EXTRARANGE)`

(You can multiply/add/subtract from the extrarange define to fit)